base.cfg

```diff
@@ -0,0 +1,115 @@
+[buildout]
+show-picked-versions = true
+extensions =
+    mr.developer
+
+parts =
+    instance
+    test
+# we use tox for testing and linting, by default
+#    code-analysis
+    coverage
+    test-coverage
+    createcoverage
+    releaser
```

Comment:
We don't need releaser in here. When releasing I use the fullrelease script from the coredev buildout.

i18ndude is not needed either: i18ndude is also run from coredev to go over all packages to get the messages. Could still be useful for checking though.

Personally, I have both these tools installed globally as well.

Reply:
@mauritsvanrees ok, i will be remove this